windows_service/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - windows_service
 
+## 3.0.0 / 2021-01-25
+
+* [Changed] Report UNKNOWN instead of CRITICAL if no Windows service matches the …. See [#8171](https://github.com/DataDog/integrations-core/pull/8171).
```

Comment:
```suggestion
* [Changed] Report UNKNOWN instead of CRITICAL if no Windows service matches the query. See [#8171](https://github.com/DataDog/integrations-core/pull/8171).
```
Title must have been too long